mk_verificator/tests/system/test_ntp_sync.py

```diff
@@ -1,89 +1,52 @@
-    print "Trying to draw a table with the results..."
-    global h, m, s, h_gauge, m_gauge, s_gauge
-
-    tab = tt.Texttable()
-    tab.set_chars(['-', '|', '+', '-'])
-    tab.set_cols_align(["c", "c", "c", "c"])
-    tab.set_cols_valign(["c", "c", "c", "c"])
-    tab.add_row(["Node", "Time", "AVG time", "Result"])
+    data = {}
+    node_times_list = []
+    hour = 0
+    minute = 0
+    second = 0
+    hour_gauge = 0
+    minute_gauge = 0
+    second_gauge = 5
+    divisor = 0
 
-    fail_tab = tt.Texttable()
-    fail_tab.set_chars(['-', '|', '+', '-'])
-    fail_tab.set_cols_align(["c", "c", "c", "c"])
-    fail_tab.set_cols_valign(["c", "c", "c", "c"])
-    fail_tab.add_row(["Node", "Time", "AVG time", "Result"])
-
-    for node in data_to_draw:
-        ntime = data_to_draw.get(node)
-
-        n_time = "{}h {}m {}s".format(ntime[0],
-                                      ntime[1],
-                                      ntime[2])
-        tgauge = "{}h {}m {}s".format(h, m, s)
-
-        if (int(ntime[0]) - h) != h_gauge:
-            fail_tab.add_row([node, n_time, tgauge, "FAILED"])
-        elif (int(ntime[1]) - m) != m_gauge:
-            fail_tab.add_row([node, n_time, tgauge, "FAILED"])
-        elif (int(ntime[2]) - s) > s_gauge:
-            # TODO: add correct verification for seconds difference
-            fail_tab.add_row([node, n_time, tgauge, "FAILED"])
-        else:
-            tab.add_row([node, n_time, tgauge, "+"])
+    fail = {}
 
-    print "Table with results:"
-    print tab.draw()
+    nodes_info = local_salt_client.cmd(group, 'cmd.run', ['date +"%H %M %S"'])
 
-    print "\nTable with FAILED results:"
-    print fail_tab.draw()
-
-
-def main():
-    local = client.LocalClient()
-
-    try:
-        print "Trying to obtain nodes time..."
-        nodes_info = local.cmd('*', 'cmd.run', ['date +"%H %M %S"'])
-    except Exception as e:
-        print e
-
-    print "Filling up buffer with data..."
     for node, time in nodes_info.iteritems():
         node_times = time.split(' ')
-        data_to_draw[node] = node_times
+        data[node] = node_times
         node_times_list.append(node_times)
 
-    count_avg_time(node_times_list)
-    draw_results_table(data_to_draw)
-    print "\nDONE"
-
+    for time_list in node_times_list:
+        hour += int(time_list[0])
+        minute += int(time_list[1])
+        second += int(time_list[2])
+        divisor += 1
+    hour = hour / divisor
+    minute = minute / divisor
+    second = second / divisor
+
+    for node in data:
+        ntime = data.get(node)
+
+        if (int(ntime[0]) - hour) != hour_gauge:
+            fail[node] = "{}h {}m {}s".format(ntime[0], ntime[1], ntime[2])
+        elif (int(ntime[1]) - minute) != minute_gauge:
+            fail[node] = "{}h {}m {}s".format(ntime[0], ntime[1], ntime[2])
+        elif (int(ntime[2]) - second) > second_gauge:
+            # TODO: add correct verification for seconds difference
```

Comment:
just for the future - please specify your name for TODO like:
TODO (vkhalin) text here...
because in this way it's more simpler to find who posted this comment (you may use git history, but that's additional actions from your side ;-) ) and directly ask them about them vision how to implement some feature.